pkg/project/version_bundle.go

```diff
@@ -15,6 +15,14 @@ func NewVersionBundle() versionbundle.Bundle {
 					"https://github.com/giantswarm/aws-operator/pull/2059",
 				},
 			},
+			{
+				Component:   "kubelet",
```

Comment:
Not sure is this correct component, please advise

Reply:
@giantswarm/team-ludacris I was long thinking about a qualified white list to get the components of our changelogs straight. I am also always wondering what is actually the component to use. What about having something for this in `fmt`?

Reply:
Yes I think it's accurate and I'm fine with it for now. But it could also be `kubernetes` or even `aws-operator`.

Reply:
This is still something we need to define. Right now there is no clear consensus. Might need discussion.